util/util.go

```diff
@@ -0,0 +1,15 @@
+package util
+
+import "os"
+
+type UtilOperations interface {
```

Comment:
Not sure this interface is neccessary unless you're going to define a struct that implements it?

Reply:
Should we rename this file to env, and contain all the env var name constants in here too?